examples/embed/widget.py

```diff
@@ -103,7 +103,7 @@ def update_pyramid(self):
 pop = Population()
 pop.render()
 
-tag = embed.autoload_server(pop.layout)
+tag = embed.autoload_server(session_id=pop.session.id, pop.layout)
```

Comment:
though, I just realized in two of the most common cases this doesn't arise.
 * with output_notebook, we autoload_server for you: https://github.com/bokeh/bokeh/pull/3325/files#diff-ab5e8987987503b1248b3f5c707e056dR270
 * if you were writing an app that embeds an autoload, you only have to pull_session if you are going to modify each session; the more common case is to just call `autoload_server` and let it generate a session_id for you, with no session object seen